### PR TITLE
Desktop adaptation

### PR DESCRIPTION
**Pull request by Danie12345**
Finished full adaptation of project (up to the moment) for the desktop version.

Changed and added many styles, as well as restructured the html.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -54,41 +56,46 @@ <h3 class="h3-first">I'm a Software Developer</h3>
...
-                    </a>
+                <li class="grid-item multi-post-stories-outer grid-item-top-pc">
+                    <a href="" class="multi-post-image"><img src="Icons/img-placeholder1.png" alt=""></a>
+                    <div class="multi-post-stories multi-post-stories-column">
+                        <h3 class="multi-post-title h3-second">Multi-Post Stories</h3>
+                        <p class="multi-post-text">
+                            A daily selection of privately personalized reads; no accounts or sign-ups required; 
+                            has been the industry's standard dummy text ever since the 1500s, when an unknown 
+                            printer took a standard dummy text.
+                        </p>
```
Comment (thecodechaser) [inline index.html]: - [x] You did a great job on the styles but for this project, you are supposed to stick to the [Figma ](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471)template as much as possible. Please reduce the gap between (image and details) these elements and make sure it matches the design on the template.

### Your design

image

### Figma design

image

File: index.html [hunk 2]
```diff
@@ -245,7 +252,6 @@ <h3 class="project-title">Professional Art Printing Data</h3>
                         </span>
                     </a>
                 </li>
-                <li class="no-project"></li>
             </ul>
         </section>
         <section class="about-me decorative-background2">
```
Comment (thecodechaser) [inline index.html:256]: - [x] Please make sure the about section background image is vertically centered as on the design. For that, you need to reduce the width of about section details.

### Your design

image

### Figma design

image

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @Danie12345👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- GitHub flow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by sja-thedude — APPROVED** [review]
## Project Approved 👍 

Hi @Danie12345, This is @sja-thedude your code reviewer.

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

giphy

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**